### PR TITLE
Initial setup for examples airflow dags with Marquez

### PR DESCRIPTION
**Pull request by ankitcha**
1. Added dockerfile for airflow with installed marquez plugin
2. added compose file & scripts required for running docker-compose stack
3. added couple of examples dags to begin with for bash & dummy operators

### CODE REVIEW

File: docker/init-db.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (ankitcha) [inline docker/init-db.sh:1]: copied from https://github.com/MarquezProject/marquez/blob/master/docker/init-db.sh - we could perhaps improve this flow

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM puckel/docker-airflow:1.10.6
```
Comment (wslulciuc) [inline Dockerfile:1]: Should we use the official docker image distributed by Apache for Airflow instead? See: [`apache/airflow`](https://hub.docker.com/r/apache/airflow)

Comment (wslulciuc) [reply Dockerfile:1]: Looks like using `puckel/docker-airflow` might be the more "stable" image to us. Disregard my comment above :)

**Review by wslulciuc — CHANGES_REQUESTED** [review]
We'll want to avoid checking in `.pyc` files into version control. I'd copy [`.gitignore`](https://github.com/MarquezProject/marquez-python/blob/master/.gitignore) from `marquez-python`

**Review by wslulciuc — APPROVED** [review]
Really amazing to finally get this much requested example started! Great work, @ankitcha 👍

**Pull request merged**